website/static/js/pages/registration-edit-page.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+require('css/registrations.css');
+
+var ko = require('knockout');
+var $ = require('jquery');
+
+var $osf = require('js/osfHelpers');
+var registrationUtils = require('js/registrationUtils');
+var RegistrationEditor = registrationUtils.RegistrationEditor;
+
+var ctx = window.contextVars;
+var node = window.contextVars.node;
+
+$(function() {
+
+    var draftEditor = new RegistrationEditor({
+        schemas: '/api/v1/project/schemas/',
```

Comment:
Building URLs inside code this way is generally discouraged; at farthest move we'd prefer to pass in the URLs from a template (using contextVars etc). Among other things, that allows us to use helpers like `api_url_for` (to aid refactoring / changeover to new API)